openchs-server-api/src/main/java/org/openchs/web/OperationalModulesController.java

```diff
@@ -41,10 +48,15 @@ public JsonObject getModules() {
                 .stream()
                 .map(FormMappingContract::fromFormMapping)
                 .collect(Collectors.toList());
+        List<AddressLevelTypeContract> addressLevelTypeContracts = addressLevelTypeRepository.getAllLowestAddressLevel()
+                .stream()
+                .map(AddressLevelTypeContract::fromAddressLevelType)
+                .collect(Collectors.toList());
         return new JsonObject()
                 .with("subjectTypes", subjectTypeRepository.findAllOperational())
                 .with("programs", programRepository.findAllOperational())
                 .with("encounterTypes", encounterTypeRepository.findAllOperational())
-                .with("formMappings", formMappingContracts);
+                .with("formMappings", formMappingContracts)
+                .with("lowestLevelAddresses", addressLevelTypeContracts);
```

Comment:
@hithacker should we allow this? This API is being called in multiple places, hence needs to be lightweight. Lowest address levels can run upto hundreds, and potentially much more. 

I would suggest keeping a different API for addresses.

Reply:
@vinayvenu I think the variable name is misleading. It's address level types and not address levels.